docs/modules/ROOT/pages/context-population.adoc

```diff
@@ -0,0 +1,50 @@
+= Context population
+
+The request context can be populated with a number of different values to influence the behaviour of the Neo4j GraphQL Library.
+
+== `driverConfig`
+
```

Comment:
Is this topic missing its content? I see that there are other like that, so I'm supposing this is going to be filled later on?